.github/workflows/sonar-step-springboot-analyze.yml

```diff
@@ -112,7 +128,7 @@ jobs:
         run: ${{ inputs.PRE_SCAN_COMMANDS }}
 
       - name: Run Sonar
-        run: ./mvnw -ntp initialize sonar:sonar -Dsonar.host.url=${{ secrets.SONAR_HOST_URL }} -Dsonar.login=${{ secrets.SONAR_TOKEN }} -Dsonar.qualitygate.wait=${{ inputs.CHECK_QUALITY_GATE }}
+        run: ./mvnw -ntp initialize sonar:sonar --settings ${{inputs.MAVEN_USER_HOME}}/settings.xml -Dsonar.host.url=${{ secrets.SONAR_HOST_URL }} -Dsonar.login=${{ secrets.SONAR_TOKEN }} -Dsonar.qualitygate.wait=${{ inputs.CHECK_QUALITY_GATE }}
```

Comment:
se skippiamo lo step di maven, qua sicuramente va poi gestito che poi non si trova la cartella

Reply:
Ho fatto che crei il settings.xml di default https://github.com/zupit-it/pipeline-templates/pull/125/commits/ac7f376ee3bbc3dff43eefb01e0ca4f6b155cdc9

Reply:
E ho testato il funzionamenti qui: https://github.com/zupit-it/apss-medico-competente-backend/pull/411